### PR TITLE
Login with `u` key

### PR DESCRIPTION
**Pull request by yskmt**
I don't think we have to change any data structures after logging in.

### CODE REVIEW

**Comment by michael-lazar** [comment]
Cool! I don't like the ``+`` key though.. how about ``u``?

**Comment by yskmt** [comment]
yeah `u` is fine. I just realized that PRAW log you out when you fail to log in. I will come up with a fix.

File: rtv/page.py [hunk 1]
```diff
@@ -226,17 +226,43 @@ def downvote(self):
...
+        username = self.prompt_input('Enter username: ')
+        password = self.prompt_input('Enter password: ', hide=True)
+
+        if (username == '') or (password == ''):
+            curses.flash()
+            return
+        try:
+            self.reddit.login(username, password)
+            
+        except praw.errors.InvalidUserPass:
```
Comment (Brobin) [inline rtv/page.py]: Is this thrown only when the password is wrong, or if either the username doesn't exist or password is wrong?

Comment (yskmt) [inline rtv/page.py]: @Brobin Yeah I think that checks username/password combination. If the username does not exists, then it always fails.

**Comment by yskmt** [comment]
The only way I can think of to not get logged out is to first check username/password with dummy log in. Then upon successful dummy log in, RTV tries to log in with the real internal variable.

Also, I changed the log in behavior so that if there is no username specified, then RTV logs out.

**Comment by michael-lazar** [comment]
How about this:

- If you are logged in, pressing ``u`` will prompt ``Log out? (y/n):`` and then log out
- If you are not logged in, pressing ``u`` will prompt for username/password

So to switch accounts, you need to press ``u`` twice, once to log out and once to log back in. That way the user is protected against accidentally logging out.

**Comment by Brobin** [comment]
This seems like a good way to handle this. :+1:

**Comment by michael-lazar** [comment]
I cleaned the code up a little bit and merged this. Thanks!

I think that the hidden password prompt is still a little weird right now. Re-sizing the terminal while entering the password breaks the login. It would be great if you could find a cleaner solution for this.

**Pull request merged**